components/library/wxwidgets-3/wxwidgets-3.p5m

```diff
@@ -26,11 +26,11 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 # Common issue. Ignore dependency on libGL.so.1 and depend on service/opengl/ogl-select.
 <transform file -> add pkg.depend.bypass-generate libGL\.so\.1>
 
-link path=usr/bin/$(MACH64)/wx-config \
+link path=usr/bin/$(MACH64)/wx-config-3 \
     target=../../lib/$(MACH64)/wx/config/gtk3-unicode-3.1
 link path=usr/bin/$(MACH64)/wxrc target=wxrc-3.1
 file path=usr/bin/$(MACH64)/wxrc-3.1
-link path=usr/bin/wx-config target=../lib/wx/config/gtk3-unicode-3.1
+link path=usr/bin/wx-config-3 target=../lib/wx/config/gtk3-unicode-3.1
 link path=usr/bin/wxrc target=wxrc-3.1
```

Comment:
The same is true for wxrc. 
Try to install wxWidgets-3 and old wxwidgets together.

Reply:
With the latest fix I was able to install `wxwidgets` and `wxwidgets-3` together. `pkg fix` on both packages is clean.

Reply:
OK. Have you compiled anything against it (and used compiled app)? 
I'm trying to build updated audacity, but it has own issues (unrelated to wxwidgets).

Reply:
Right. I think someone on the oi-discuss ml was building (older version of) Filezilla with wxwidgets, might be a good candidate. Will have a look during the weekend.